migrations/README.rst

```diff
@@ -227,6 +255,27 @@ If you face this error, it means that the :code:`pepys-import` repository should
 
 ------------
 
+.. code-block:: bash
+
+    Database tables are not found! (Hint: Did you initialise the DataStore?)
+
+If you migrated your **SQLite** database and started to see this message, it would have been because of the extra tables created by Alembic. When a table with Geometry column is included in migration,
+please check the SQLite database if there are any temporary tables (with *tmp* keyword) such as :code:`idx_Geometries_tmp_geometry_node` etc. If there is any, please drop these tables. After that,
+please open your migration script and add :code:`spatial_index=False` argument to the Geometry column. An example is as follows:
```

Comment:
Is this something that the developer does when creating the migration script? Or something that the end user might need to do? I'd rather not have the end user having to list and drop tables in the SQLite database.

Can we just set spatial_index=False all the time?

Reply:
This problem might occur only if there is a wrongly implemented migration script. So, it's something for developers. If they see that there are extra redundant tables, it points out that new migration script has something wrong.

> Can we just set spatial_index=False all the time?
I thought the same but I think it wouldn't work when we tried to use `BaseSpatiaLite.metadata.create_all()` because it should be created for once. (The first file in migration scripts does this job if migration is used to create all tables)

Reply:
Ok, could you clarify in the docs here that this is for developers only, and shows that the migration script is wrong.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/427/commits/56fb60e75509ae1a0c815f32667b0062e02e9d02